hopsworks-IT/src/test/ruby/spec/helpers/dataset_helper.rb

```diff
@@ -36,6 +36,8 @@
  DAMAGES OR  OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 =end
+require 'rest-client'
```

Comment:
@ErmiasG shall we close this PR? There is another one for the same JIRA but that other one is missing this line